org.openjdk.jmc.console.ext.agent/src/main/java/org/openjdk/jmc/console/ext/agent/ui/AgentUi.java

```diff
@@ -125,11 +125,10 @@ public void handleEvent(Event event) {
 	private boolean loadAgent(String agentJar, String xmlPath, String pid) {
 		try {
 			VirtualMachine vm = VirtualMachine.attach(pid);
-			if (xmlPath == null || xmlPath == ENTER_PATH_MSG) {
-				File tempFile = materialize(TEMP_DIR_NAME, NO_EVENT_PROBES_XML, AgentUi.class);
-				xmlPath = tempFile.getPath();
-			}
-			vm.loadAgent(agentJar, xmlPath);
+			if (xmlPath == null || xmlPath.equals(ENTER_PATH_MSG)) {
+				vm.loadAgent(agentJar);
+			} else {
+				vm.loadAgent(agentJar, xmlPath);			}
```

Comment:
This is trivial, but please add a line feed

Reply:
Fixed https://github.com/rh-jmc-team/jmc-agent-plugin/pull/15/commits/7eed387d291207679e91066889c8cc1b906099cd